lib/parsers/log-parser.js

```diff
@@ -76,10 +85,27 @@ export default class LogParser extends Parser {
 
       match = line.match(ERROR_PATTERN)
       if (match) {
+        let outMessageText
         const lineNumber = match[2] ? parseInt(match[2], 10) : undefined
+        // Check if the error is "Undefined control sequence"
+        let errMessageText = match[4]
```

Comment:
What do you think about renaming this to `errorMessage`?

Reply:
Yes. I am a fan of explicit names. `errorMessage` it is!